tap_jira/streams.py

```diff
@@ -272,7 +272,7 @@ def sync(self):
 COMPONENTS = Stream("components", ["id"], indirect_stream=True)
 ISSUES = Issues("issues", ["id"])
 ISSUE_COMMENTS = Stream("issue_comments", ["id"], indirect_stream=True)
-ISSUE_TRANSITIONS = Stream("issue_transitions", ["id"],
+ISSUE_TRANSITIONS = Stream("issue_transitions", ["id","issueId"], # Composite primary key
```

Comment:
@KrisPersonal  Both the fields `id` and `issue_id` are already present in the schema so both fields should be there in the target side table and only 'id' was a primary key before.
Previously, customers might have missed data due to an incorrect primary key but after this change customers will see all the records.

Also, we are not much familiar with the target side and not sure about the exact effect of the primary key change.
@dmosorast @cosimon  can you please help us to answer what kind of effect it will have on the target side?